web/ui/karma.conf.js

```diff
@@ -87,7 +87,6 @@ module.exports = function(config) {
       // specify a common output directory
       dir: 'test/results/coverage',
       reporters: [
-        { type: 'html', subdir: 'html' },   // use for local dev review
```

Comment:
FYI - If you change the `test` task to use Chrome or Firefox for one of the `browsers` then you can get the interactive report which show you coverage detail.  The coverage for baseFactory is super good (generally >90). The coverage for servicesFactory is so so  (in the 60s). The big outliers are `mergeArray`, and the getters for "hosts" and "addresses"